class ObjectNotFoundError(Exception):
    pass


class ObjectAlreadyExistsError(Exception):
    pass
